matter_server/server/__main__.py

```diff
@@ -72,6 +72,7 @@ def main() -> None:
     coloredlogs.install(level=args.log_level.upper())
     if not logging.getLogger().isEnabledFor(logging.DEBUG):
         logging.getLogger("chip").setLevel(logging.WARNING)
+        logging.getLogger("PersistentStorage").setLevel(logging.WARNING)
```

Comment:
This is a rather unrelated change and should be a separate commit (that also has the advantage that it appears in changelogs etc.)

Reply:
Yes it should be a seperated commit indeed, I'll put it in a seperate commit

Reply:
but well in the end this will be a squash commit so it doesnt matter after all. I'll add it to the PR desciption